src/main/java/it/gov/pagopa/payhub/activities/activity/debtposition/SaveDebtPositionActivity.java

```diff
@@ -0,0 +1,21 @@
+package it.gov.pagopa.payhub.activities.activity.debtposition;
+
+import it.gov.pagopa.payhub.activities.dto.debtposition.DebtPositionDTO;
+
+
+/**
+ * Service class responsible for saving debt positions.
+ */
+public interface SaveDebtPositionActivity {
+
+    /**
+     * Saves a new debt position.
+     * <p>
+     * This method processes the provided {@link DebtPositionDTO} and ensures it is persisted
+     * using the underlying data storage mechanisms.
+     * </p>
+     *
+     * @param debtPosition the {@link DebtPositionDTO} containing the details of the debt position to be saved
+     */
+    void saveDebtPosition(DebtPositionDTO debtPosition);
```

Comment:
let's return the saved DTO: it should contains the stored ids, if this class become a real activity, its result would be cached, and stored id lost